h2o-algos/src/test/java/water/network/SSLEncryptionTest.java

```diff
@@ -0,0 +1,93 @@
+package water.network;
+
+import hex.Distribution;
+import hex.Model;
+import hex.ModelMetricsBinomial;
+import hex.deeplearning.DeepLearning;
+import hex.deeplearning.DeepLearningModel;
+import hex.tree.gbm.GBM;
+import hex.tree.gbm.GBMModel;
+import org.junit.Assert;
+import org.junit.Ignore;
+import water.DKV;
+import water.TestUtil;
+import water.fvec.Frame;
+import water.fvec.Vec;
+import water.util.Log;
+import water.util.MathUtils;
+
+import java.util.Date;
+
+import static hex.Distribution.Family.bernoulli;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * This class is used to capture TCP packets while training a model
+ * The result is then used to check if SSL encryption is working properly
+ */
+@Ignore
```

Comment:
Added the Ignore just in case so no one adds Test annotations there - I run that class in `testSSL.sh`